components/gcp-click-to-deploy/src/Utils.ts

```diff
@@ -1,5 +1,8 @@
 import * as bcrypt from 'bcryptjs';
 
+const GITHUB_BASE_URL =
+    'https://raw.githubusercontent.com/kubeflow/kubeflow/master/';
+
```

Comment:
Config file on kubeflow master (or other branch) might not compatible with kfctl backend.
Shall we bake v0.5 & v0.6 config files into image during build?

Reply:
Currently, that's how it's done for the 0.5.0 compatible deployment but Jeremy had a comment to try to fetch it from GitHub. Pulling from master is probably problematic but we can easily point to a branch for each version to ensure that we get a consistent file that should be compatible with the version in question.